packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md

```diff
@@ -27,10 +29,33 @@ You will need the following things for this guide:
 
 ## Installation
 
-Include the following script tag in your target web page:
+### Using the npm module
+
+Using our npm module is a good choice if:
+
+- You have a build system in place where you manage dependencies with npm.
+- You do not want to load scripts directly from third party sites.
+
+To install [@okta/okta-auth-js](https://www.npmjs.com/package/@okta/okta-auth-js):
+
+```bash
+# Run this command in your project root folder.
+# yarn
+yarn add @okta/okta-auth-js
+
+# npm
+npm install --save @okta/okta-auth-js
+```
+
+### Using script tag in your target web page
 
-``` html
-<script src="https://ok1static.oktacdn.com/assets/js/sdk/okta-auth-js/2.0.1/okta-auth-js.min.js" type="text/javascript"></script>
+If you're using the JS on a web page from the browser, you can copy the `node_modules/@okta/okta-auth-js/dist` contents to publicly hosted directory, and include a reference to the `okta-auth-js.min.js` file in a `<script>` tag.  
```

Comment:
contents to publicly hosted directory, and include.... -> contents to a publicly hosted directory and include....